### PR TITLE
*: migrate test framework to testify

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Issue Number: Ref #4813.

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

File: pkg/assertutil/assertutil.go [hunk 1]
```diff
@@ -22,20 +22,14 @@ type Checker struct {
 }
 
 // NewChecker creates Checker with FailNow function.
-func NewChecker(failNow func()) *Checker {
```
Comment (JmPotato) [inline pkg/assertutil/assertutil.go:25]: Why remove this arg?

Comment (rleungx) [reply pkg/assertutil/assertutil.go:25]: Once we remove all `pingcap/check` dependencies, we can change it back.

File: server/schedule/labeler/rule_test.go [hunk 2]
```diff
@@ -17,46 +17,44 @@ package labeler
...
 
 	// test legal rule with ttl
 	label.TTL = "10h10m10s10ms"
 	err = label.checkAndAdjustExpire()
-	c.Assert(err, IsNil)
-	c.Assert(len(label.StartAt) > 0, IsTrue)
-	c.Assert(label.expireBefore(time.Now().Add(time.Hour)), IsFalse)
-	c.Assert(label.expireBefore(time.Now().Add(24*time.Hour)), IsTrue)
+	re.NoError(err)
+	re.True(len(label.StartAt) > 0)
```
Comment (lhy1024) [inline server/schedule/labeler/rule_test.go:45]: how about `re.Greater`

**Review by JmPotato — APPROVED** [review]

**Pull request merged**